packages/jest/src/create-serializer.js

```diff
@@ -36,10 +36,11 @@ function getNodes(node, nodes = []) {
   return nodes
 }
 
-function copyProps(src, target) {
-  return Object.defineProperties(src, {
-    ...Object.getOwnPropertyDescriptors(target)
-  })
+function copyProps(target, source) {
```

Comment:
the previous names were confusing for me, usually when copying props from one object to another they are called the other way around (how I've adjusted them here)